src/phonebook/views.py

```diff
@@ -59,16 +68,56 @@ def dump_phonebook(self, poc: bool) -> list[DectRegistration]:
             }
             if poc:
                 # POC member, include extra info
+                ipei = None
+                if dect.ipei:
+                    ipei = dectutil.format_ipei(dect.ipei[0], dect.ipei[1])
                 entry.update(
                     {
                         "activation_code": dect.activation_code,
                         "publish_in_phonebook": dect.publish_in_phonebook,
+                        "ipei": ipei,
                     },
                 )
             phonebook.append(entry)
         return phonebook
 
 
+@method_decorator(csrf_exempt, name="dispatch")
+class DectUpdateIPEI(
```

Comment:
Can we rename this to ApiDectUpdateIPEI or something? so it is clear it is an api view and not an HTML view